client/src/core/client/admin/routes/Configure/sections/ModerationPhases/ModerationPhasesConfigContainer.tsx

```diff
@@ -50,26 +49,6 @@ const ModerationPhasesBox: FunctionComponent<{ children: React.ReactNode }> = ({
 const ModerationPhasesConfigContainer: FunctionComponent<Props> = ({
   settings,
 }) => {
-  const [{ dsaFeaturesEnabled }] =
-    useLocal<ModerationPhasesConfigContainer_local>(graphql`
-      fragment ModerationPhasesConfigContainer_local on Local {
-        dsaFeaturesEnabled
-      }
-    `);
-
-  if (dsaFeaturesEnabled) {
-    return (
-      <ModerationPhasesBox>
-        <Localized id="configure-moderationPhases-dsaEnabled">
-          <CallOut color="error" fullWidth>
-            You currently have DSA features enabled. External moderation is not
-            currently DSA compliant, and so is disabled.
-          </CallOut>
-        </Localized>
-      </ModerationPhasesBox>
-    );
-  }
-
   return (
```

Comment:
Don't we want to also update the copy here in the admin configuration to call out that if DSA is enabled, then a rejection reason is required to be sent through?